### ISSUE TITLE
brew install failing (master & develop). "No such file or directory - src/github.com/ethereum/go-ethereum/cmd/mist/assets"

### ISSUE THREAD

**Issue by simondlr** [ethereum/go-ethereum#1331]
Mist has been removed 5 days ago (https://github.com/ethereum/go-ethereum/commit/42a14b8a09d812dbd7d1b05f145e6881376f320b#diff-dfff0a7fa1a55c8c1a4966c19f6da452), but it seems it still wants to include it? Brew install fails.

On master & develop.

```
(...)
github.com/huin/goupnp
github.com/huin/goupnp/dcps/internetgateway1
github.com/huin/goupnp/dcps/internetgateway2
github.com/ethereum/go-ethereum/p2p/nat
github.com/ethereum/go-ethereum/crypto
github.com/ethereum/go-ethereum/p2p/discover
github.com/ethereum/go-ethereum/cmd/bootnode
Error: No such file or directory - src/github.com/ethereum/go-ethereum/cmd/mist/assets
```

**Comment by tgerring** [comment]
Pinging buildmaster @caktux :)

**Comment by caktux** [comment]
should be fixed in ethereum/homebrew-ethereum@b8e90ce although it wasn't failing on the build server.. just make sure you `brew update` :)

**Comment by simondlr** [comment]
Great! It works smoothly now. Thanks @caktux!
